tests/mcproto/types/test_nbt.py

```diff
@@ -1095,8 +499,8 @@ def check_equality(self: object, other: object) -> bool:
                     return False
             return True
         if isinstance(self, list):
-            self = cast("list[Any]", self)
-            other = cast("list[Any]", other)
+            self = cast(List[Any], self)
+            other = cast(List[Any], other)
```

Comment:
Isn't it what you asked me to do here : https://github.com/py-mine/mcproto/pull/285#discussion_r1602201776

Reply:
yes, it is. I asked to use the new `list[Any]`, instead of the deprecated `typing.List` types. This diff indicates a change from `"list[Any]"` back to `typing.List[Any]`